### PR TITLE
Fix content/en/docs/tutorials/stateless-application/guestbook.md

### PR DESCRIPTION
**Pull request by jailton**
Update "Deploying PHP Guestbook application with MongoDB" Tutorial to fix the following issues:
- Ordered lists with wrong indexes
- Wrong port of mongo service example
- Wrong output of "kubectl get service frontend" on LoadBalancer Service Type example.
- Formatting issues

Fix: #26765
Fix: #26732 

Preview: https://deploy-preview-26766--kubernetes-io-master-staging.netlify.app/docs/tutorials/stateless-application/guestbook/

/kind cleanup 
/language en

### CODE REVIEW

File: content/en/docs/tutorials/stateless-application/guestbook.md [hunk 1]
```diff
@@ -102,9 +103,9 @@ kubectl apply -f ./content/en/examples/application/guestbook/mongo-service.yaml
       The response should be similar to this:
 
       ```shell
-      NAME           TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
-      kubernetes     ClusterIP   10.0.0.1     <none>        443/TCP    1m
-      mongo          ClusterIP   10.0.0.151   <none>        6379/TCP   8s
+      NAME         TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)     AGE
+      kubernetes   ClusterIP   10.96.0.1      <none>        443/TCP     7m7s
+      mongo        ClusterIP   10.96.30.147   <none>        27017/TCP   2m17s
```
Comment (sftim) [inline content/en/docs/tutorials/stateless-application/guestbook.md:108]: I think these are the only two lines that really need to change @jailton - the others look OK to keep.

Comment (jailton) [reply content/en/docs/tutorials/stateless-application/guestbook.md:108]: @sftim I change the other lines to:
- maintain the pod name and IP consistency on all examples;
- moving comments lines to fix issue #26765 (ordered lists with wrong indexes);
- fixing the type name at line 219.

**Pull request closed without merging**